rust/backend/daemon/src/server.rs

```diff
@@ -48,20 +47,9 @@ impl Ziofa for ZiofaImpl {
     }
 
     async fn list_processes(&self, _: Request<()>) -> Result<Response<ProcessList>, Status> {
-        // dummy data
-        let response = ProcessList {
-            processes: vec![
-                Process {
-                    pid: 1,
-                    package: "systemd".to_string(),
-                },
-                Process {
-                    pid: 1741231,
-                    package: "com.example.org".to_string(),
-                },
-            ],
-        };
-        Ok(Response::new(response))
+        // TODO: Error handling
+        let processes = list_processes().unwrap();
```

Comment:
the .unwrap could be replaced with ? i think, the client can then decide what todo on error...

Reply:
For that you need your own error type, that implements the `Into<Status>` trait. For an example you can look into `counter.rs`.

Reply:
Done. Fits you?